docs/plugins/aws/awsCompileScheduledEvents.md

```diff
@@ -0,0 +1,13 @@
+# awsCompileScheduledEvents
+
+This plugins compiles the function schedule event to to a CloudFormation resource.
+
+## How it works
+
+`awsCompileScheduleEvents` hooks into the `compileEvents` hook of the [deploy](/docs/plugins/core/deploy.md) plugin.
```

Comment:
`awsCompileScheduledEvents` hooks into the [`deploy:compileEvents`](/docs/plugins/core/deploy.md) hook.